drivers/vulkan/vulkan_context.cpp

```diff
@@ -2247,13 +2249,13 @@ RID VulkanContext::local_device_create() {
 			/*ppEnabledLayerNames */ nullptr,
 			/*enabledExtensionCount */ enabled_extension_count,
 			/*ppEnabledExtensionNames */ (const char *const *)extension_names,
-			/*pEnabledFeatures */ &physical_device_features, // If specific features are required, pass them in here
+			/*pEnabledFeatures */ &physical_device_features, // If specific features are required, pass them in here.
 		};
 		err = vkCreateDevice(gpu, &sdevice, nullptr, &ld.device);
 		ERR_FAIL_COND_V(err, RID());
 	}
 
-	{ //create graphics queue
+	{ /* Create graphics queue. */
```

Comment:
Prefer `//` to `/* */` when writing comments.

Reply:
Is there somewhere that has the exact cases of when to use `//` over `/* */`? It seemed that a lot of places in the code would use `//` for titles of code like the above example but other places would use  `/* */`.

Reply:
https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html

Reply:
Thank you that doc is very helpful. I was getting lost in all the different kinds of documentation guidelines last night. Most are for the document pages and not for the code documentation so I couldn't find the rules for comments.

Reply:
Should `//` be used for single line comments inside of if and else statements?

Reply:
When in doubt, use `//`.

Reply:
Is ensuring comments have periods and capitalization necessary? The code style guidelines appear to have periods and capitalization.

Reply:
Yes. There is existing code without periods and capitalization and it's an ongoing effort to improve this.

Reply:
Ok perfect I'll make sure I fix any I come across.